xml/FinanceType-category.xml

```diff
@@ -9,127 +9,147 @@
         <url>http://www.oecd.org/dac/stats/dacandcrscodelists.htm</url>
     </metadata>
     <codelist-items>
-        <codelist-item>
+        <codelist-item status="active">
+            <code>0</code>
+            <name>
+                <narrative>NON FLOW ITEMS</narrative>
```

Comment:
I have a feeling this code is deliberately excluded from the IATI replicated codelist. @bill-anderson can confirm or deny.

Reply:
@andylolz @PetyaKangalova is this above comment here holding up the merge or can it still go?

Reply:
Aha – here’s why I think this is deliberately excluded: https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/16#issuecomment-51773027

Reply:
I have no idea whether that means they should or shouldn’t be included. I shall leave it with you to decide!

Reply:
I would agree that finance type '0- non flow items' does not fit within the activity standard as it does not describe type of finance at activity level.  It is much wider than that. I would propose that we do **not** replicate- waiting for confirmation from @bill-anderson so  @samuele-mattiuzzo please hold off merging until then.

Reply:
I just talked to @bill-anderson  and the view is that we keep this code in, the reason being that we replicate all 'OECD DAC' codelists exactly as they are in the table provided by the DAC.  @samuele-mattiuzzo  this means we can continue merging this request.

However, what will be useful is to add a note in the Finance Type and Finance Type (Category) to note that non-flow items are not activity specific and we do not expect the codes to be used when reporting finance types in the IATI activity standard.

Reply:
Not really sure about the note idea.

If it’s important enough that it’s a problem, then you can either add a rule to the ruleset, or leave the codes out.

I guess in general, I think it’s best to avoid situations where the standard strongly advises something, but doesn’t enforce in any way.

Reply:
re: "or leave the codes out."
As already mentioned the standard replicates third-party code lists. This is a well-established principle.

Reply:
Hi @bill-anderson – yes, but cf. this comment from 2014 @bill-anderson:
https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/16#issuecomment-51773027